metricbeat/mb/registry.go

```diff
@@ -0,0 +1,141 @@
+package mb
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/elastic/beats/libbeat/logp"
+)
+
+const initialSize = 10 // initialSize specifies the initial size of the Register.
```

Comment:
No, this value is about the registry size and number of modules being registered.